scripts/gen_proto.sh

```diff
@@ -37,3 +37,10 @@ protoc -I proto/ \
     --go_out=plugins=grpc:./pkg/pb/custom_detectorspb --go_opt=paths=source_relative \
     --validate_out="lang=go,paths=source_relative:./pkg/pb/custom_detectorspb" \
     proto/custom_detectors.proto
+protoc -I proto/ \
+    -I ${GOPATH}/src \
+    -I /usr/local/include \
+    -I ${GOPATH}/src/github.com/envoyproxy/protoc-gen-validate \
+    --go_out=plugins=grpc:./pkg/detectors/trufflehogenterprisescanner/scannerpb --go_opt=paths=source_relative \
+    --validate_out="lang=go,paths=source_relative:./pkg/detectors/trufflehogenterprisescanner/scannerpb" \
+    proto/api.proto
```

Comment:
this is to generate the heartbeart protos

Reply:
**Nit (optional):** The filename `api.proto` doesn't sound like it's being used for detector validation but I'm not sure what a better name would be. Could it be added to `detectors.proto` or maybe `detectors_validation.proto`?